nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/groups/StandardProcessGroup.java

```diff
@@ -4022,7 +4027,7 @@ private VersionedFlowSynchronizationContext createGroupSynchronizationContext(fi
     }
 
     @Override
-    public void verifyCanSaveToFlowRegistry(final String registryId, final String bucketId, final String flowId, final String saveAction) {
+    public void verifyCanSaveToFlowRegistry(final String registryId, final String branch, final String bucketId, final String flowId, final String saveAction) {
```

Comment:
Should not the newly created `FlowLocation` used? If I am correct, branch, bucket id and flow id covers just the same set of info as `FlowLocation`s state covers.